src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -420,7 +420,10 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
 
                 bindingContext.ModelState.TryAddModelError(
                     modelStateKey,
-                    Resources.FormatModelBinding_MissingBindRequiredMember(propertyName));
+                    string.Format(
+                        CultureInfo.CurrentCulture,
+                        propertyExplorer.Metadata.ValidationErrorMessages.MissingBindRequiredValueResource(),
+                        propertyName));
```

Comment:
Defining funcs of the proper arity is better than demanding that you give us a resource string with the right number of parameters in it. :+1: